src/components/menu-bar/menu-bar.jsx

```diff
@@ -66,6 +66,13 @@ import languageIcon from '../language-selector/language-icon.svg';
 
 import scratchLogo from './scratch-logo.svg';
 
+const messages = defineMessages({
+    confirmNav: {
+        id: 'gui.sharedMessages.confirmNewWithoutSaving',
```

Comment:
I thought about that, but thought it was not particular to the menu bar -- it could apply to the editor as a whole. Does that make sense? Not sure what the convention is.

Reply:
Changed to gui,menuBar.confirmNewWithoutSaving